ihmc-footstep-planning/src/main/generated-java/us/ihmc/footstepPlanning/AStarBodyPathPlannerParametersBasics.java

```diff
@@ -370,4 +371,89 @@ default void setSmootherGradientThresholdToTerminate(double smootherGradientThre
    {
       set(AStarBodyPathPlannerParameters.smootherGradientThresholdToTerminate, smootherGradientThresholdToTerminate);
    }
+
+   default void set(AStarBodyPathPlannerParametersPacket packet)
+   {
+         setCheckForCollisions(packet.getCheckForCollisions());
+         setComputeSurfaceNormalCost(packet.getComputeSurfaceNormalCost());
+         setComputeTraversibility(packet.getComputeTraversibility());
+         setPerformSmoothing(packet.getPerformSmoothing());
+
+      if (packet.getRollCostWeight() != -1.0)
+         setRollCostWeight(packet.getRollCostWeight());
+      if (packet.getRollCostDeadband() != -1.0)
+         setRollCostDeadband(packet.getRollCostDeadband());
+      if (packet.getMaxPenalizedRollAngle() != -1.0)
+         setMaxPenalizedRollAngle(packet.getMaxPenalizedRollAngle());
+      if (packet.getSnapRadius() != -1.0)
+         setSnapRadius(packet.getSnapRadius());
+      if (packet.getMinSnapHeightThreshold() != -1.0)
+         setMinSnapHeightThreshold(packet.getMinSnapHeightThreshold());
+      if (packet.getInclineCostWeight() != -1.0)
+         setInclineCostWeight(packet.getInclineCostWeight());
+      if (packet.getInclineCostDeadband() != -1.0)
+         setInclineCostDeadband(packet.getInclineCostDeadband());
+      if (packet.getMaxIncline() != -1.0)
```

Comment:
Maybe have a static final variable `DEFAULT_NULL_VALUE = -1.0` for all the -1.0's

Reply:
Good call, I'm adding this to the message...